meta-xt-prod-aos-rcar-driver-domain/conf/layer.conf

```diff
@@ -0,0 +1,13 @@
+# We have a conf and classes directory, add to BBPATH
+BBPATH .= ":${LAYERDIR}"
+
+# We have a packages directory, add to BBFILES
+BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
+            ${LAYERDIR}/recipes-*/*/*.bbappend"
+
+BBFILE_COLLECTIONS += "xt-prod-aos-driver-domain"
+BBFILE_PATTERN_xt-prod-aos-driver-domain := "^${LAYERDIR}/"
+BBFILE_PRIORITY_xt-prod-aos-driver-domain = "6"
```

Comment:
Maybe it's better to increase priority to 7 or 8 to allow overwriting of original recipes.

Reply:
done